front/app/containers/IdeasShow/GoBackButton.tsx

```diff
@@ -16,6 +22,7 @@ interface Props {
 const GoBackButton = memo(({ projectId, className, insideModal }: Props) => {
   const project = useProject({ projectId });
   const locale = useLocale();
+  const { windowWidth } = useWindowSize();
```

Comment:
I would recommend you use `useBreakpoint` here instead of this hook for performance reasons.
Here is the [full discussion](https://github.com/CitizenLabDotCo/cl2-component-library/pull/168) on why.

Reply:
💯 less code! Thanks for the recommendation.